client/coral-framework/graphql/queries/myCommentHistory.graphql

```diff
@@ -0,0 +1,9 @@
+query myCommentHistory {
```

Comment:
Are you referring to the fact that the `fragments` are in another folder? That's because `fragments` should be unique. That's why we need to import them everytime we need them. Also, I think that breaking the queries into different files is cleaner and clearer. 

Another thing is that: Fragments are no longer suported using `createFragment` https://github.com/apollostack/react-apollo/issues/435 (I created this because the Docs are not updated)



We no longer can send the fragment like this.